arlo_server/election_settings.py

```diff
@@ -0,0 +1,63 @@
+from flask import jsonify, request
+from jsonschema import validate
+
+from arlo_server import app, db
+from arlo_server.auth import require_audit_admin_for_organization
+from arlo_server.models import Election
+
+from util.jsonschema import nullable, Obj, Str, Bool, Int
+
+
+PUT_ELECTION_SETTINGS_REQUEST_SCHEMA = Obj(
+    electionName=Str,
+    online=Bool,
```

Comment:
Any reason these two aren't nullable as well?

Reply:
They aren’t nullable in the database.